internal/github/checkUnsigned.go

```diff
@@ -96,3 +99,33 @@ func hasUnsignedCommits(ctx context.Context, githubClient *github.Client, pr *gi
 	}
 	return true, nil
 }
+
+// CheckAndComment checks for the specific comment from a specific account and posts a comment if it doesn't exist.
+func CheckAndComment(ctx context.Context, client *github.Client, owner, repo string, prNumber int) error {
+	commentAuthor := "ChiaAutomation"
+	commentBody := "Your commits are not signed and our branch protection rules require signed commits. For more information on how to create signed commits, please visit this page: https://docs.chia.net/contribution/using-github/#setup-commit-signing. Please use the button towards the bottom of the page to close this pull request and open a new one with signed commits."
+	comments, _, err := client.Issues.ListComments(ctx, owner, repo, prNumber, nil)
+	if err != nil {
+		return fmt.Errorf("error fetching comments: %v", err)
+	}
+
+	// Check if the comment already exists
+	for _, comment := range comments {
+		if comment.GetUser().GetLogin() == commentAuthor && strings.EqualFold(comment.GetBody(), commentBody) {
+			slogs.Logr.Info("Comment already exists", "repo", repo, "PR", prNumber)
+			return nil
+		}
+	}
+
+	// Post the comment if it doesn't exist
+	comment := &github.IssueComment{
+		Body: github.String(commentBody),
+	}
+	slogs.Logr.Info("Comment", "comment", comment)
```

Comment:
```suggestion
	slogs.Logr.Info("Adding unsigned commit comment", "owner", owner, "repo", repo, "PR", prNumber, "comment", comment)
```
Adds context to the log

Reply:
I'm not sure the comment really needs to be a part of it either. If you know the version of the app, you can look up in the code what the comment would have been (or in the PR timeline.) But I left that in my suggestion.

Reply:
I am deleting this. This was for testing at one point actually.